### DISCUSSION TITLE
Modal woking once and then automatically closing immediately

### DISCUSSION THREAD

**Discussion by Gardylooo** [tailwindlabs/headlessui#1358] [Help]
I was having an issue with the modal in my project, so I tried to run the basic modal example. Turns out that does not also work don't know how to fix that, it does not show any error. The model only once works when I do a hard reset to the server, and when it works the modal automatically closes immediately 

Here is the link to the project on [codesandbox](https://codesandbox.io/s/happy-dirac-8rwsqf)

**Comment by RobinMalfait** [comment]
It's a React 18 issue, checkout #681
